tools/mason/MasonExternal.chpl

```diff
@@ -297,8 +291,11 @@ proc getSpkgInfo(spec: string, ref dependencies: list(string)): unmanaged Toml t
     var version = specFields[1];
     var compiler = specFields[2];
 
-    if spkgInstalled(spec) {
-      const spkgPath = getSpkgPath(spec);
+    // Remove variants from spec
+    var simpleSpec = pkgName + '@' + version + '%' + compiler;
+
```

Comment:
@ben-albrecht  I was working on [#15421 ](https://github.com/chapel-lang/chapel/issues/15421) for an error message on missing version. Is this where the error should be added ?